OpenRA.Editor/Form1.cs

```diff
@@ -238,7 +238,9 @@ void PrepareMapResources(ModData modData, Map map)
 					if (rsi != null && rsi.EditorPalette != null && rsi.EditorPalette.Contains("terrain"))
 						templatePalette = palette;
 
-					var template = RenderUtils.RenderActor(info, tileset, templatePalette);
+					var race = Program.Rules.Actors["world"].Traits.WithInterface<CountryInfo>().First().Race;
```

Comment:
The problem is that the editor ignores the buildings / units completely in d2k because it doesn't understand our sequences code.  It is a waste of time to implement this properly because it isn't used in practice, and will be removed in the hopefully-not-too-distant future.